spec/javascripts/controllers/containers/container_topology_controller_spec.js

```diff
@@ -27,4 +27,11 @@ describe('containerTopologyController', function() {
             expect($scope.kinds).not.toBe(undefined);
         });
     });
+
+    describe('kinds contain all expected kinds', function() {
+        it('in all main objects', function() {
+               expect(Object.keys($scope.kinds).length).toBeGreaterThan(7);
```

Comment:
> Thanks! And I do promise to do the automatic thing sooner rather than later so that you don't have to put up with my obsessive whitespace fixation ;)

:+1: 

can you find a volunteer to merge this please?